src/js/colorpicker-component.js

```diff
@@ -227,6 +227,7 @@ Colorpicker.prototype = {
     val = val || this.color.toString(this.format, false);
     if (this.input !== false) {
       this.input.prop('value', val);
+      this.input.trigger('change');
```

Comment:
as requested in other of your PR, the regeneration of the dist folder is required